helm/korifi/templates/global-deny-apps-policy.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.experimental.securityGroups.enabled }}
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: deny-apps-policy
```

Comment:
How about setting the network policy name to something that describes that app egress should be denied, for example `deny-apps-egress-policy`? That name should be also reflected into the file name as well